package.json

```diff
@@ -50,7 +45,10 @@
 					"*.ts"
 				],
 				"rules": {
-					"@typescript-eslint/no-unnecessary-condition": "error",
+					"ava/assertion-arguments": "off",
+					"@typescript-eslint/dot-notation": "off",
+					"@typescript-eslint/no-non-null-assertion": "off",
+					"@typescript-eslint/no-non-null-asserted-optional-chain": "off",
```

Comment:
Unfortunate but most tests use `regex.test()` and it won't be a problem if the result is `regex.test(undefined)`